samples/quarkus/pet-store/pom.xml

```diff
@@ -0,0 +1,141 @@
+<?xml version="1.0"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>com.amazon.quarkus.demo</groupId>
```

Comment:
Should be com.amazonaws.serverless.sample and the artifact serverless-quarkus-example to follow the same pattern we use in the [regular samples](https://github.com/awslabs/aws-serverless-java-container/blob/master/samples/jersey/pet-store/pom.xml).

Reply:
updated.